includes/rest/class-inbox.php

```diff
@@ -331,61 +331,6 @@ public static function shared_inbox_post_parameters() {
 		return $params;
 	}
 
-	/**
-	 * Handles "Reaction" requests
-	 *
-	 * @param  array $object  The activity-object
-	 * @param  int   $user_id The id of the local blog-user
-	 */
-	public static function handle_reaction( $object, $user_id ) {
```

Comment:
Are we deleting this because it wasn't working anyway? Or I guess we'd already removed the hook? I think we mention "reactions" in the readme too, we should probably remove those references as well.

Reply:
It was not in use and I do not know if it really works. I think we should give the comments feature a bit more love in one of the next major versions.